freshmaker/handlers/koji/rebuild_eus_base_images.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021  Red Hat, Inc.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import re
+
+from freshmaker import conf
+from freshmaker.events import ManualRebuildWithAdvisoryEvent
+from freshmaker.lightblue import LightBlue
+from freshmaker.errata import Errata
+from .rebuild_images_on_rpm_advisory_change import RebuildImagesOnRPMAdvisoryChange
+
+
+class RebuildEUSBaseImages(RebuildImagesOnRPMAdvisoryChange):
+    """
+    Rebuilds Extended Life Support base images
+    """
+
+    name = 'RebuildEUSBaseImages'
+
+    def can_handle(self, event):
+        if not super().can_handle(event):
+            return False
+
+        # If 'content_sets' of the event doesn't exist yet, create and fill it
+        if self.event.content_sets is None:
+            errata = Errata()
+            self._set_event_content_sets(errata, int(event.search_key))
+
+        # Check if any of the content sets is EUS content set
+        for content_set in self.event.content_sets:
+            if re.match(r'.*-eus($|\W)', content_set):
+                return True
+
+        return False
```

Comment:
Optional: Just an alternative syntax that is shorter if you prefer this. You can disregard it if you prefer your syntax.
```suggestion
        return any(re.match(r'.+-eus-rpms', c) for c in self.event.content_sets)
```